src/transformers/models/gpt_neo/modeling_gpt_neo.py

```diff
@@ -130,7 +130,151 @@ def load_tf_weights_in_gpt_neo(model, config, gpt_neo_checkpoint_path):
     return model
 
 
-class GPTNeoSelfAttention(nn.Module):
+class AttentionMixin:
```

Comment:
If it's relative to GPT Neo, could it have it in the name?
```suggestion
class GPTNeoAttentionMixin:
```